magefile.go

```diff
@@ -496,7 +496,11 @@ func Config() {
 
 // ControlProto generates pkg/agent/control/proto module.
 func ControlProto() error {
-	return sh.RunV("protoc", "--go_out=plugins=grpc:.", "control.proto")
+	return sh.RunV(
+		"protoc",
+		"--go_out=internal/pkg/agent/control/cproto", "--go_opt=paths=source_relative",
+		"--go-grpc_out=internal/pkg/agent/control/cproto", "--go-grpc_opt=paths=source_relative",
+		"control.proto")
 }
```

Comment:
We should add this to the `Update` as a dependencies to ensure that the generated files are in sync. This will will be called in our CI and check if any file get updated with git status.